### ISSUE TITLE
Doesn't seems to search anything

### ISSUE THREAD

**Issue by adocampo** [tranxuanthang/lrcget#78]
I've readed on reddit the source is musixmatch, which it's not ideal for me because it seems there is not much of what I listen, but after searching for some artists and songs in musixmatch with lyrics, I cannot download anything. 

Well, the screen capture will make a better idea what I'm talking about.
![image](https://github.com/tranxuanthang/lrcget/assets/2266612/1bf4aa64-7b2b-44ed-9e24-5e8a79bed809)

**Comment by adocampo** [comment]
Hummm , just readed LRCGet obtains the lyrics from https://lrclib.net/ and there, there isn't that song
https://lrclib.net/search/Soziedad%20Alkoholika

but you say "Lrclib in turn fetches and caches the lyrics from various lyrics sources (such as MXM)." if that MXM means MusiX Match, shouldn't be on the results?

**Comment by tranxuanthang** [comment]
There is some information that might provide you with more insight into how LRCLIB works:

- LRCLIB only obtains new lyrics with metadata from the `/api/get` API (equivalent to the "Download All Lyrics" button in LRCGET), not from the `/api/search` API.
- LRCLIB used to directly scrape lyrics from third-party sources during `/api/get` requests, display the results, and cache them. Due to performance reasons, it no longer works that way. Currently, any missed (not found) `/api/get` requests will immediately return a 404 error, and the track metadata is pushed to a processing queue to retrieve the lyrics from third-party sources later.
- Basically, if you missed a lot of lyrics using the "Download All Lyrics" feature, you might find some more by simply rerunning it a few hours later.
- Due to high traffic to the `/api/get` API recently, it takes an excessively long time (hours or even more than a day) for a queue job to be processed. I'm currently finding a way to optimize the processing queue to make it as fast as possible.
